Library/Formula/gcc.rb

```diff
@@ -210,6 +216,17 @@ def install
       rmdir lib64
       prefix.install_symlink "lib" => "lib64"
     end
+    
+    if OS.linux?
+      p = @link.parent
+      @link.delete
+      p.delete
+      crts = Pathname.new "#{lib}/gcc/x86_64-unknown-linux-gnu/#{version}"
+      Formula['glibc'].lib.children.select {|p| p.basename.to_s =~ /^crt.\.o$/ }.collect {|p| p.relative_path_from crts}.each do |p|
+        crts.install_symlink p 
+      end
+    end
```

Comment:
I understand that this patch allows GCC to find crt1.o, crti.o and crtn.o by linking them into a directory that GCC searches by default. What's the purpose of the above symlink from `"#{HOMEBREW_PREFIX}/lib"` to `"#{prefix}/x86_64-unknown-linux-gnu/bin"`?